locale/gu.yml

```diff
@@ -16,75 +16,98 @@ language-abbreviation: "gu"
   NOTE: "નોંધ"
     # ✅ @179priyasoni
     # ✅ @aadii0408
+    # ✅ @shahkv95
   INFO: "માહિતી"
     # ✅ @179priyasoni
     # ✅ @aadii0408
+    # ✅ @shahkv95
   IDEA: "વિચાર"
     # ✅ @179priyasoni
     # ✅ @aadii0408
-  DEBUG: ""
+    # ✅ @shahkv95
+  DEBUG: "ખામી-સુધારવી"
```

Comment:
I need to update the documentation to reflect this - if the first translator does not enter anything into the initial space, we will keep it blank. This is so we can better track the progression of reviews. 👍  I will update this for you as the documentation did not describe how to do this correctly.

Reply:
@shahkv95 it seems I do not have permissions to push to your repo, so I cannot make my changes in this PR. Forgive my work around 👍